include/tables/object_table.h

```diff
@@ -0,0 +1,652 @@
+/* 0x12A */ DEFINE_OBJECT(object_bombiwa, OBJECT_BOMBIWA)
+/* 0x12B */ DEFINE_OBJECT(object_hintnuts, OBJECT_HINTNUTS)
+/* 0x12C */ DEFINE_OBJECT(object_rs, OBJECT_RS)
+/* 0x12D */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_12D)
+/* 0x12E */ DEFINE_OBJECT(object_gla, OBJECT_GLA)
+/* 0x12F */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_12F)
+/* 0x130 */ DEFINE_OBJECT(object_geldb, OBJECT_GELDB)
+/* 0x131 */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_131)
+/* 0x132 */ DEFINE_OBJECT(object_dog, OBJECT_DOG)
+/* 0x133 */ DEFINE_OBJECT(object_kibako2, OBJECT_KIBAKO2)
+/* 0x134 */ DEFINE_OBJECT(object_dns, OBJECT_DNS)
+/* 0x135 */ DEFINE_OBJECT(object_dnk, OBJECT_DNK)
+/* 0x136 */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_136)
+/* 0x137 */ DEFINE_OBJECT(object_gi_insect, OBJECT_GI_INSECT)
+/* 0x138 */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_138)
+/* 0x139 */ DEFINE_OBJECT(object_gi_ghost, OBJECT_GI_GHOST)
+/* 0x13A */ DEFINE_OBJECT(object_gi_soul, OBJECT_GI_SOUL)
+/* 0x13B */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_13B)
+/* 0x13C */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_13C)
+/* 0x13D */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_13D)
+/* 0x13E */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_13E)
+/* 0x13F */ DEFINE_OBJECT(object_gi_rupy, OBJECT_GI_RUPY)
+/* 0x140 */ DEFINE_OBJECT(object_mu, OBJECT_MU)
+/* 0x141 */ DEFINE_OBJECT(object_wf, OBJECT_WF)
+/* 0x142 */ DEFINE_OBJECT(object_skb, OBJECT_SKB)
+/* 0x143 */ DEFINE_OBJECT(object_gs, OBJECT_GS)
+/* 0x144 */ DEFINE_OBJECT(object_ps, OBJECT_PS)
+/* 0x145 */ DEFINE_OBJECT(object_omoya_obj, OBJECT_OMOYA_OBJ)
+/* 0x146 */ DEFINE_OBJECT(object_cow, OBJECT_COW)
+/* 0x147 */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_147)
+/* 0x148 */ DEFINE_OBJECT(object_gi_sword_1, OBJECT_GI_SWORD_1)
+/* 0x149 */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_149)
+/* 0x14A */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_14A)
+/* 0x14B */ DEFINE_OBJECT(object_zl4, OBJECT_ZL4)
+/* 0x14C */ DEFINE_OBJECT(object_link_goron, OBJECT_LINK_GORON)
+/* 0x14D */ DEFINE_OBJECT(object_link_zora, OBJECT_LINK_ZORA)
+/* 0x14E */ DEFINE_OBJECT(object_grasshopper, OBJECT_GRASSHOPPER)
+/* 0x14F */ DEFINE_OBJECT(object_boyo, OBJECT_BOYO)
+/* 0x150 */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_150)
+/* 0x151 */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_151)
+/* 0x152 */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_152)
+/* 0x153 */ DEFINE_OBJECT(object_fwall, OBJECT_FWALL)
+/* 0x154 */ DEFINE_OBJECT(object_link_nuts, OBJECT_LINK_NUTS)
+/* 0x155 */ DEFINE_OBJECT(object_jso, OBJECT_JSO)
+/* 0x156 */ DEFINE_OBJECT(object_knight, OBJECT_KNIGHT)
+/* 0x157 */ DEFINE_OBJECT(object_icicle, OBJECT_ICICLE)
+/* 0x158 */ DEFINE_OBJECT(object_spdweb, OBJECT_SPDWEB)
+/* 0x159 */ DEFINE_OBJECT_UNSET(OBJECT_UNSET_159)
+/* 0x15A */ DEFINE_OBJECT(object_boss01, OBJECT_BOSS01)
+/* 0x15B */ DEFINE_OBJECT(object_boss02, OBJECT_BOSS02)
+/* 0x15C */ DEFINE_OBJECT(object_boss03, OBJECT_BOSS03)
+/* 0x15D */ DEFINE_OBJECT(object_boss04, OBJECT_BOSS04)
+/* 0x15E */ DEFINE_OBJECT(object_boss05, OBJECT_BOSS05)
+/* 0x15F */ DEFINE_OBJECT_NULL(object_boss07, OBJECT_UNSET_15F)
```

Comment:
I like OoT's approach here with justi suffixing `UNUSED` and having a 0 size comment, since it gives a little more context.
For example
`/* 0x00FA */ DEFINE_OBJECT_NULL(object_md, OBJECT_MD_UNUSED) // unused duplicate with size 0`